{
  "name": "Untrusted data passed to external API",
  "description": "Data provided remotely is used in this external API without sanitization, which could be a security risk.",
  "cwe": "security external/cwe/cwe-020",
  "cwe-description": null,
  "language": "python",
  "checker-language": "ql",
  "example": null,
  "loc": 12,
  "test": [],
  "branches": 0,
  "apis": 7
}